docs/installation/create-new-project.md

```diff
@@ -50,7 +50,7 @@ the scaffolding phase.
 | Test Runner | Middleware |
 | --- | --- |
 | Jest | [`@neutrinojs/jest`](../packages/jest) |
-| Karma | [`@neutrinojs/karma`](../pakages/karma) |
+| Karma | [`@neutrinojs/karma`](../packages/karma) |
```

Comment:
Yeah I know! 😭

Reply:
Fwiw mkdocs strict mode should avoid some of these in the future (though it will help less with the absolute neutrinojs.org URLs since it doesn't check those... though I guess we could make the build process convert them prior... then would solve both problems... I'll have a think).